libs/src/services/creatorNode/index.js

```diff
@@ -392,9 +392,16 @@ class CreatorNode {
       return resp.data
     } catch (e) {
       if (e.response && e.response.data && e.response.data.error) {
+        console.error(`Server returned error: [${e.response.status.toString()}] ${e.response.data.error}`)
```

Comment:
Sweet, TY for doing this! Two thoughts - since we're consoling the same error we're throwing here and down below, might be nice to pull it out to a string constant and then just `console.error(errString) throwNewError(errString)`. Could be also be nice to extract this whole bit into a helper function `handleCreatorRequestError` that we can use both in `_makeRequest` and `_upload`?

Reply:
@piazz good call. i made a new helper function